cocotb/share/makefiles/Makefile.sim

```diff
@@ -42,8 +42,37 @@ regression                Run simulator when dependencies have changes
 clean                     Remove build and simulation artefacts
 help                      This help text
 
-Make Variables
---------------
+GPI
+---
+GPI_EXTRA                 Extra libraries to load at runtime (comma-separated)
+
+Cocotb
+------
+TOPLEVEL                  Instance in the hierarchy to use as the DUT
+RANDOM_SEED               Random seed, to recreate a previous test stimulus
+COCOTB_ANSI_OUTPUT        Force cocotb to print or not print in color
+COCOTB_REDUCED_LOG_FMT    Display log lines shorter
+COCOTB_ATTACH             Pause time value in seconds before the simulator start
+COCOTB_ENABLE_PROFILING   Performance analysis of the Python portion of cocotb
+COCOTB_HOOKS              Comma-separated module list to be executed before test
+COCOTB_LOG_LEVEL          Default logging level (default INFO)
+COCOTB_RESOLVE_X          How to resolve X, Z, U, W on integer conversion
+MEMCHECK                  HTTP port to use for debugging Python memory usage
+
+Regression Manager
+------------------
```

Comment:
Since this is a subcategory of cocotb:
```suggestion
~~~~~~~~~~~~~~~~~~
```

Reply:
Oh, I forget this is not rST.

Reply:
I have made these changes in ``building.rst`` where it *is* rST.